pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -652,7 +652,6 @@ def test_td_rfloordiv_numeric_series(self):
         msg = "Invalid dtype"
         with pytest.raises(TypeError, match=msg):
             # Deprecated GH#19761, enforced GH#29797
```

Comment:
could probably remove this comment as well or does it adds value?

Reply:
either way i guess, the historical reference doesnt hurt